src/aap_eda/api/serializers/activation.py

```diff
@@ -297,7 +367,8 @@ def to_representation(self, activation):
             else None
         )
         activation_instances = models.RulebookProcess.objects.filter(
-            activation_id=activation.id
+            parent_type=ProcessParentType.ACTIVATION,
+            id=activation.id,
```

Comment:
Are you using the activation id to filter the id of the rulebook process. This would return wrong results. I think this line doesn't need to be changed. You want the rulebook_processes related with the activation id.